packages/angular/src/constants.ts

```diff
@@ -3,8 +3,3 @@ export const ANGULAR_ROUTING_OP = 'ui.angular.routing';
 export const ANGULAR_INIT_OP = 'ui.angular.init';
 
 export const ANGULAR_OP = 'ui.angular';
-
-/**
- * Minimum Angular version this SDK supports
- */
-export const ANGULAR_MINIMUM_VERSION = 10;
```

Comment:
I decided to move this constant back to `sdk.ts` so that we only have to duplicate that file. It's only used in in this file anyway.

Reply:
I don't understand this, don't we symlink everything?

Reply:
Everything but `sdk.ts`. I should have explained this in the PR description, sorry. `sdk.ts` differs in two ways:
1. It sets the SDK name to `sentry.javascript.angular-ivy`
2. It adjusts the Angular version check with
  - the minimum Angular version being v12
  - a warning message that's better worded for angular-ivy